bitrepository-alarm-service/src/main/java/org/bitrepository/alarm/alarmservice/WebAlarm.java

```diff
@@ -43,7 +43,7 @@ public class WebAlarm {
 
     public WebAlarm(Alarm alarm) {
         collectionID = alarm.getCollectionID() == null ? "" : alarm.getCollectionID();
-        fileID = alarm.getFileID() == null ? "" : alarm.getFileID();
+        fileID = (alarm.getFileID() == null) ? "" : alarm.getFileID();
```

Comment:
After rebasing quite many changes that didn’t come from my hand are now included in the review, which is probably confusing.